### PR TITLE
Proto encapsulate

### PR DESCRIPTION
**Pull request by kelunik**

### CODE REVIEW

**Comment by LeanderK** [comment]
it would be cool if in the future you would explain a bit what happened in the protos, so the reviewer does not have to guess based on commit messages and code

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/transformers/ExperimentTransformer.java [hunk 1]
```diff
@@ -16,19 +17,28 @@
...
+ * Handles transformations of experiments from and to the protocol buffer messages.
+ *
+ * @author Leander K.
  * @author Marcel Hollderbach
+ * @author Niklas Keller
  */
 public class ExperimentTransformer extends AbstractTransformer {
+    private static Integer toInteger(int value) {
+        return Integer.newBuilder().setValue(value).build();
+    }
```
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/database/transformers/ExperimentTransformer.java]: i would move this in AbstractTransformer

**Comment by kelunik** [comment]
I even linked the appropriate commit in the spec. What exactly should I explain here?

**Comment by LeanderK** [comment]
no, in the description of the pull-request...its not that big of an issue ;)

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java [hunk 2]
```diff
@@ -5,6 +5,10 @@
 import spark.Request;
 
 public class RequestUtil {
+    private RequestUtil() {
+        // utility class
+    }
```
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java]: i would write some javadoc ;) just to justify the private constructor

Comment (kelunik) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java]: Usually documentation for private members isn't generated anyway and it's justified by the comment inside?

Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java]: utility class are still not always static. And they are not generated when you export to html, but the IDE still shows them. But i was talking more about the class-javadoc. something like:
`\** bla bla contains static methods to help with Sparkjava-requests....`

Comment (kelunik) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java]: Oh, Javadoc on the class is actually miss, right. Added it.

**Pull request merged**